camayoc/tests/qpc/api/v1/utils.py

```diff
@@ -157,11 +161,13 @@ def wait_until_state(scanjob, timeout=120, state='completed'):
 
     while (not scanjob.status() or not scanjob.status()
             == state) and timeout > 0:
-        if state == 'stopped' and scanjob.status() in QPC_SCAN_TERMINAL_STATES:
+        if state in stopped_states and scanjob.status() in stopped_states:
```

Comment:
We are calling `scanjob.status()` many times here which is going a read request. I think we can do better and check the status once at the beginning of the while loop.

Reply:
I think I'd like to check once at top and once after the call to time.sleep